tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb

```diff
@@ -485,42 +447,31 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "color=get_color(signal,labels)\n",
-    "show_figure(signal,'audio with speaker labels',color)\n",
-    "display(Audio(signal,rate=16000))"
+    "transcription_path_to_file = f\"{ROOT}/asr_with_diartranscript_with_speaker_labels/an4_diarize_test.txt\"\n",
```

Comment:
`/` is missing after asr_with_diar folder. 
Correct path
 `{ROOT}/asr_with_diar/transcript_with_speaker_labels/an4_diarize_test.txt`

Reply:
fixed